ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1227,8 +1227,7 @@ private void LoadDefinedNames(Workbook workbook)
                         }
                         else
                         {
-                            string sheetName, sheetArea;
-                            ParseReference(area, out sheetName, out sheetArea);
+                            ParseReference(area, out String sheetName, out String sheetArea);
```

Comment:
Just a side note: later this method may be redefined to return a tuple ```(string shetName, string sheetArea)```.

Reply:
Yes, we should review each of 7.2's additions and see where we can use it. I know of some other places where we can use tuples too.